src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/AutoInsert/IOnAutoInsertProvider.cs

```diff
@@ -8,5 +8,5 @@ namespace Microsoft.CodeAnalysis.Razor.AutoInsert;
 
 internal interface IOnAutoInsertProvider : IOnAutoInsertTriggerCharacterProvider
 {
-    public VSInternalDocumentOnAutoInsertResponseItem? TryResolveInsertion(Position position, RazorCodeDocument codeDocument, bool enableAutoClosingTags);
+    public bool TryResolveInsertion(Position position, RazorCodeDocument codeDocument, bool enableAutoClosingTags, out VSInternalDocumentOnAutoInsertResponseItem? autoInsertEdit);
```

Comment:
Needs `[NotNullWhen(true)]` on the out parameter

Reply:
Thanks, yes, caught and fixed a few of these.